specs/zrok.yml

```diff
@@ -985,7 +985,7 @@ definitions:
       oauthProvider:
         type: string
         enum: ["github", "google"]
-      oauthEmailDomains:
+      oauthEmailAddressPatterns:
```

Comment:
Shoot. This is probably the _only_ place where we _don't_ want to change this... because it will break API compatibility. An old client that just sends an email address will still work, unless we change the API spec.